test/EmptyScriptRoot/host.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Could not check in an empty folder. Our host code automatically generates this file if not exist. So I decided to check in this so that the empty folder structure exist and can be used for `DotNetIsolated_PlaceholderMiss_EmptyScriptRoot` test.